src/main/resources/static/js/global-variables.js

```diff
@@ -69,6 +69,22 @@ function getFrontendServiceBackEndPath() {
     return frontendServiceBackEndPath;
 }
 
+function getSubscriptionNameRegex() {
+    return subscriptionNameRegex;
+}
+
+function getNotificationMetaRegex() {
+    return notificationMetaRegex;
+}
+
+function getUsernameMaxDisplayLength() {
+    if (usernameMaxDisplayLength < 4) {
+        // Minimum leangth is 4 due to ellipsis being 3 long
+        return 4;
+    }
+    return usernameMaxDisplayLength;
+}
+
 // End   ## getters and setters
 
 // Start ## getters and setters status handling
```

Comment:
I think "getters and setters" should be removed from comment. It is more confusing than not to have it here in my opinion.